### DISCUSSION TITLE
Multiple Inheritance from PolymorphicModel?

### DISCUSSION THREAD

**Discussion by MiltosD** [jazzband/django-polymorphic#707] [Q&A]
class A(PolymorphicModel)

class B(PolymorphicModel)

class C(B)

class D(A, C)

When trying to make migrations for class **D**, I get the following
**_The field 'polymorphic_ctype' from parent model 'A' clashes with the field 'polymorphic_ctype' from parent model 'B'._**

Is there any way to differentiate between those two?

**Comment by JackAtOmenApps** [comment]
Is there more to the error? Post the entire stack trace. I use the same setup in one of my apps, and have not encountered this.

**Comment by claudioap** [comment]
```python
class Subscriptible(PolymorphicModel):
    subscribers = djm.ManyToManyField(User)

class Postable(PolymorphicModel):
    author = djm.ForeignKey(settings.AUTH_USER_MODEL)

class Answer(Postable, Subscriptible):
    pass
```

`Answer: (models.E005) The field 'polymorphic_ctype' from parent model 'postable' clashes with the field 'polymorphic_ctype' from parent model 'subscriptible'.`

Issue #429 could be relevant

**Comment by bckohan** [comment]
@MiltosD  Multiple inheritance from concrete models does work in Django - you have to manually specify and deconflict the primary key field names.

django-polymorphic does not yet support multiple inheritance. There are a few issues to resolve (namely the ctype conflict you highlight) - but it is on the list. For now, you would need to make all but one parent abstract.
